pkgs/tools/misc/undocker/default.nix

```diff
@@ -1,25 +1,37 @@
 { lib
 , buildGoModule
-, fetchFromSourcehut
+, fetchFromGitea
+, git
+, gnumake
 }:
-buildGoModule rec {
-  pname = "undocker";
-  version = "1.0.4";
-
-  src = fetchFromSourcehut {
-    owner = "~motiejus";
-    repo = pname;
+let
+  version = "1.2.1";
+  src = fetchFromGitea {
+    domain = "git.jakstys.lt";
+    owner = "motiejus";
+    repo = "undocker";
     rev = "v${version}";
-    hash = "sha256-I+pTbr1lKELyYlyHrx2gB+aeZ3/PmcePQfXu1ckhKAk=";
+    hash = "sha256-d1QKnIO+O+mBQ7TA394FZisdHP2g4iVBK/cDwKryP1M=";
+    leaveDotGit = true;
   };
+in
+buildGoModule {
+  pname = "undocker";
+  inherit version src;
+
+  nativeBuildInputs = [ git gnumake ];
+
+  buildPhase = "make VSN=v${version} undocker";
```

Comment:
So we don't need `leaveDotGit` and `git` in `nativeBuildInputs` anymore?

Reply:
Is it that important? Can we just provide a fake hash? `leaveDotGit` frequently has reproducibility issues.

Reply:
Updated.